dynamic-config/config-cc/src/main/java/org/apache/servicecomb/config/client/ConfigCenterClient.java

```diff
@@ -163,6 +163,7 @@ private void refreshMembers(MemberDiscovery memberDiscovery) {
       String configCenter = memberDiscovery.getConfigServer();
       IpPort ipPort = NetUtils.parseIpPortFromURI(configCenter);
       clientMgr.findThreadBindClientPool().runOnContext(client -> {
+        @SuppressWarnings("deprecation")
```

Comment:
Do not use deprecad api if necessary.

Reply:
vertx 3.6.3 statement the interface is deprecated, but has not provider a new interface to replace, so we have to add the annotation to paas the CI buiding